Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/LoadTestDataPatientBean.java

```diff
@@ -83,82 +113,246 @@ public void setSelectedPatient(Patient selectedPatient) {
+        return selectedPhonenumber;
+    }
+
+    public void setSelectedPhonenumber(Phonenumber selectedPhonenumber) {
+        this.selectedPhonenumber = selectedPhonenumber;
+    }
+
+    // database-methods
+    public List<Phonenumber> getPhonenumbers() {
+        if (isValidPatientId()) {
+            return loadTestDataService.getAllPhonenumbersBy(getPatientId());
+        } else {
+            return new ArrayList<>();
+        }
+    }
+
+    public boolean deletePhonenumber() {
+        if (selectedPhonenumber != null) {
+            return loadTestDataService.deletePhonenumber(selectedPhonenumber);
+        } else {
+            addPatientErrorMessages("Phonenumber have to be select to be deleted");
+        }
+        return false;
+    }
+
+    public boolean savePhonenumber() {
+        boolean actionResult = false;
+
+        if (isValidPatientId()) {
+            try {
+                Phonenumber phonenumber = readPhonenumber();
+                phonenumber.setPatient(withPatient);
+                actionResult = loadTestDataService.savePhonenumber(phonenumber);
+
+                if (actionResult) {
+                    addPatientInfoMessages("Save Phonenumber Successful: " + phonenumber.getPhonenumberId());
+                    clearInputsPhonenumbers();
+                }
+            } catch (LoadTestDataException e) {
+                logPatientError("Patient Phonenumber", e);
+            }
+        } else {
+            addPatientErrorMessages("Phonenumber cannot be save: patient does not exist.");
+        }
+        return actionResult;
+    }
+
+    public List<Address> getAddresses() {
+        if (isValidPatientId()) {
+            return loadTestDataService.getAllAddressesBy(getPatientId());
+        } else {
+            return new ArrayList<>();
+        }
+    }
+
+    public boolean deleteAddress() {
+        if (selectedAddress != null) {
+            return loadTestDataService.deleteAddress(selectedAddress);
+        } else {
+            addPatientErrorMessages("Address have to be select to be deleted");
+        }
+        return false;
+    }
+
+    public boolean saveAddress() {
+        boolean actionResult = false;
+        if (isValidPatientId()) {
+            try {
+                Address address = readAddress();
+                address.setPatient(withPatient);
+                actionResult = loadTestDataService.saveAddress(address);
+
+                if (actionResult) {
+                    addPatientInfoMessages("Save Address Successful: " + address.getAddressId());
+                    clearInputsAddresses();
+                }
+            } catch (LoadTestDataException e) {
+                logPatientError("Patient Address", e);
+            }
+        } else {
+            addPatientErrorMessages("Address cannot be save: patient does not exist.");
+        }
+
+        return actionResult;
+    }
+
+    public List<Identifier> getIdentifiers() {
+        if (isValidPatientId()) {
```

Comment:
A method should have only one return statement instead of two. Use a local variable to get the result and return the local variable. Change in other methods as well.